pypeit/core/coadd.py

```diff
@@ -228,7 +230,8 @@ def sn_weight(spectra, smask, debug=False):
 
     rms_sn = np.sqrt(np.mean(sn2)) # Root Mean S/N**2 value for all spectra
 
-    if rms_sn <= 4.0:
+    # FW: should we always using wavelength dependent weights for coadding. at least for megering orders.
```

Comment:
Wavelength dependent weights are tricky.  There can be large
variations.  Has always been safest to do order-by-order or 
spectrum-by-spectrum weighting.

Reply:
okay. removed the comments. 
For Echelle, we have options to coadd order-by-order (default) or coadd everything together.